src/sdk/resources/policies.ts

```diff
@@ -13,6 +13,11 @@ export class PoliciesResource extends PagedResource<Policy> {
 		return new PolicyTrackingGroupsResource(this.httpClient, policyId);
 	}
 
+	async schema(id: string, token?: CancellationToken): Promise<PropertySchema> {
+		const result = await this.httpClient.get(`${this.resourcePath}/${id}/schema`, null, token);
```

Comment:
Passing `null` for `params` causes `http-client` to bomb

```suggestion
		const result = await this.httpClient.get(`${this.resourcePath}/${id}/schema`, {}, token);
```

![image](https://user-images.githubusercontent.com/28901115/122119460-d7b7f580-cdee-11eb-896a-c6bf499a6c69.png)

Reply:
When attempting to destructure `params`
![image](https://user-images.githubusercontent.com/28901115/122119688-18177380-cdef-11eb-8b98-c0b1b6cb2bd2.png)

Reply:
Should be good now.